packages/venia-concept/src/util/makeMediaPath.js

```diff
@@ -1,6 +1,7 @@
 const trailingSlashRE = /\/+$/;
 const wrappingSlashRE = /^\/*(.+?)\/*$/;
-export function makePathPrepender(prefix) {
+
+function makePathPrepender(prefix) {
```

Comment:
Please note that we are no longer exporting this `makePathPrepender` higher order function (it is not used anywhere outside of this file).

Reply:
It not being used doesn't necessarily mean it shouldn't be exported (and tested). I'll approve this particular change, but in general we should export and test more units.